scripts/BeforeBuild.ps1

```diff
@@ -31,8 +31,9 @@ Set-StrictMode -Version Latest
 $ErrorActionPreference = "Stop"
 $InformationPreference = "Continue"
 
-Import-Module $PSScriptRoot\ScriptUtilities.psm1 -Force
-Import-Module $PSScriptRoot\Projects.psm1 -Force
+Import-Module -Force $PSScriptRoot\ScriptUtilities.psm1
```

Comment:
>-Force [](start = 14, length = 6)

Moving the `-Force` option to the start of the argument list improves the vertical alignment of the `Import-Module` statements.